File: WarehouseManagementWeb.Infrastructure/Identity/Models/UserSignInOutput.cs
namespace WarehouseManagementWeb.Infrastructure.Identity.Models
{
    /// <summary>
    /// Класс выходной модели прохождения аутентификации пользователя.
    /// </summary>
    public class UserSignInOutput
    {
        /// <summary>
        /// Имя пользователя.
        /// </summary>
        public string? FirstName { get; set; }

        /// <summary>
        /// Фамилия пользователя.
        /// </summary>
        public string? LastName { get; set; }

        /// <summary>
        /// Email пользователя.
        /// </summary>
        public string? Email { get; set; }

        /// <summary>
        /// Токен доступа пользователя.
        /// </summary>
        public string? AccessToken { get; set; }

        /// <summary>
        /// Токен обновления пользователя.
        /// </summary>
        public string? RefreshToken { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/auth/models/output/user-sign-in-output.ts
/**
 * Класс выходной модели прохождения аутентификации пользователя.
 */
export class UserSignInOutput {
  /**
   * Имя пользователя.
   */
  firstName: string = '';

  /**
   * Фамилия пользователя.
   */
  lastName: string = '';

  /**
   * Email пользователя.
   */
  email: string = '';

  /**
   * Токен доступа пользователя.
   */
  accessToken: string = '';

  /**
   * Токен обновления пользователя.
   */
  refreshToken: string = '';
}
